apps/user/repositories/user.repository.ts

```diff
@@ -99,19 +98,26 @@ export class UserRepository {
    * @param id
    * @returns User profile data
    */
-    async getUserPublicProfile(id: number): Promise<UserI> {
-          const queryOptions: UserQueryOptions = {
-            id
-          };
-          return this.findUserForPublicProfile(queryOptions);
-        }
+  async getUserPublicProfile(username: string): Promise<UserI> {
+
+
+    let queryOptions: UserQueryOptions = {};
```

Comment:
I think you should use `const ` instead of `let`

Reply:
Resolved